whitepaper/whitepaper.md

```diff
@@ -1112,7 +1112,7 @@ All packages have a prefix of `cactus-*` to avoid potential naming conflicts wit
 
 Naming conventions for packages:
 * cmd-* for packages that ship their own executable
-* sdk-* for packages designed to be used directly by application developers except for the Javacript SDK which is named just `sdk` for simplicity.
+* sdk-* for packages designed to be used directly by application developers except for the Javacript SDK which is named just `sdk` for simplicity. /sdk1
```

Comment:
@Leeyoungone 

```suggestion
```

(After the advancements we've done recently, this line doesn't make sense anymore so it's best to just delete it to avoid confusion)